src/components/Link.js

```diff
@@ -46,6 +46,22 @@ const InternalLink = styled(IntlLink)`
   }
 `
 
+const ExplicitLangInternalLink = styled(GatsbyLink)`
+  .is-glossary {
+    white-space: nowrap;
+  }
```

Comment:
Shouldn't need this, as it only applied to `/glossary/` links which are not in the "english only" category, if you could please remove

Reply:
👍